### ISSUE TITLE
Wrong RemoteUrls data returned when git repository contains multiple remote sections

### ISSUE THREAD

**Issue by filzrev** [kekyo/GitReader#10]
**Describe the bug**
I want to get `RemoteUrls` information with `GitReader`.
But it seems to return incorrect data if the `.git/config' file contains multiple remote sections.

**To Reproduce**
Run the following code on a git repository that contains multiple remote sections. (e.g. forked repository)

```csharp
using var repository = await Repository.Factory.OpenPrimitiveAsync("[git repository path]");
foreach(var remoteUrl in repository.RemoteUrls)
{
    Console.WriteLine($"{remoteUrl.Key}:{remoteUrl.Value}");
}
```

**Console outputs**
```
origin: https://github.com/kekyo/GitReader.git
```

**Excerpt of local `.git/config` file**
```
[remote "origin"]
	url = https://github.com/my-forked-repo/GitReader.git
	fetch = +refs/heads/*:refs/remotes/origin/*
[remote "upstream"]
	url = https://github.com/kekyo/GitReader.git
	fetch = +refs/heads/*:refs/remotes/upstream/*
```

**Expected behavior**
I'm expecting following RemoteUrls information returned that match with `git remote -v` command output..

```
origin: https://github.com/my-forked-repo/GitReader.git
upstrem: https://github.com/kekyo/GitReader.git
```

**Additional context**
The current [RepositoryAccessor::ReadRemoteReferencesAsync' method](https://github.com/kekyo/GitReader/blob/ce68b633419a8b16d642e6ea1ec3492cdbdf2584/GitReader.Core/Internal/RepositoryAccessor.cs#L143-L207) returns the following information.
- First `remote` name.
- Last `remote` url

**Comment by kekyo** [comment]
Thanks reach out! I will check it.

**Comment by kekyo** [comment]
Fixed and merged in 7bcdf07fc74b69910e9818b5c76c38974c34533a.

Thank you reported, it will be released in next version!
